src/xdocs/cmdline.xml.vm

```diff
@@ -37,7 +37,8 @@ java -D&lt;property&gt;=&lt;value&gt;  \
      [-t | --tree] [-T | --treeWithComments] [-J | --treeWithJavadoc] [-j | --javadocTree] \
      [-V | --version] [-b | --branch-matching-xpath &lt;xpathQuery&gt;] [-h | --help] \
      [-e | --exclude &lt;excludedPath&gt;] [-E | --executeIgnoredModules] [-d | --debug] \
-     [-x | --exclude-regexp &lt;excludedPathPattern&gt;] \ file...
+     [-x | --exclude-regexp &lt;excludedPathPattern&gt;] \
+     file(s) or folder(s) ...
```

Comment:
I tried to avoid any names like this, as it looks like something special.
I tried to review some CLI manual to get best practices on how others are doing this, not I didn't find any clarity.